src/main/java/services/petshop/steps/CommonSteps.java

```diff
@@ -2,43 +2,56 @@
 
 import io.restassured.response.Response;
 import net.thucydides.core.annotations.Step;
+import net.thucydides.core.annotations.Steps;
 import org.junit.Assert;
 import services.petshop.pojo.Pet;
-import services.petshop.pojo.responses.GetPetNotFoundResponse;
+import services.petshop.pojo.responses.ErrorResponse;
 import utils.methods.ReusableMethods;
 
 
 public class CommonSteps {
 
     ReusableMethods reusableMethods = new ReusableMethods();
+    @Steps
+    PostPetSteps postPetSteps;
 
     @Step("Then I assert the pet that has been retrieved is the same pet that has been created")
-    public void verifyPetEqual(Pet petCreated, Pet mypet) {
+    public void verifyPetEquals(Pet petCreated, Pet mypet) {
         Assert.assertEquals(petCreated, mypet);
     }
 
-    public Pet convertResponseToPet(Response response){
-        Pet pet = response.as(Pet.class);
-        return  pet;
+    public Pet convertResponseToPet(Response response) {
+        return (response.as(Pet.class));
     }
 
     @Step("Then I verify the error message is {2}")
     public void verifyErrMsg(Response response, int statusCode, String expectedErrorMsg) {
         reusableMethods.validateResponseStatusCode(response, statusCode);
-        GetPetNotFoundResponse petNotFoundResponse = response.as(GetPetNotFoundResponse.class);
+        ErrorResponse petNotFoundResponse = response.as(ErrorResponse.class);
         Assert.assertEquals(expectedErrorMsg, petNotFoundResponse.getMessage());
     }
 
     @Step("When I check the pet retrieved has the {1} name")
-    public void verifyPetName(Response response, String name){
+    public void verifyPetName(Response response, String name) {
         Pet pet = convertResponseToPet(response);
         Assert.assertEquals(name, pet.getName());
     }
 
     @Step("When I check the pet retrieved has the {1} status")
-    public void verifyPetStatus(Response response, String status){
+    public void verifyPetStatus(Response response, String status) {
         Pet pet = convertResponseToPet(response);
         Assert.assertEquals(status, pet.getStatus());
     }
 
+    public Pet createValidPet(String name, String status) {
+        int id = reusableMethods.generateRandomInt(10, 100);
+        Pet myPet = new Pet();
+        myPet.setId(id);
+        myPet.setName("doggie");
+        myPet.setStatus("sold");
+        postPetSteps.createNewPet(myPet);
```

Comment:
remove this line since we want to generate a valid Pet object and not create that